Packs/XMCO/Integrations/Serenety/Serenety.py

```diff
@@ -0,0 +1,254 @@
+    Converts the XMCO alert severity level ('none', 'low', 'medium', 'high', 'critical') to Cortex XSOAR alert
+    severity (1 to 4).
+
+    Args:
+        severity (str): severity as returned from the LePortail API.
+
+    Returns:
+        int: Cortex XSOAR Severity (0,5 to 4)
+    """
+    return {
+        'none': IncidentSeverity.INFO,
+        'low': IncidentSeverity.LOW,
+        'medium': IncidentSeverity.MEDIUM,
+        'high': IncidentSeverity.HIGH,
+        'critical': IncidentSeverity.CRITICAL
+    }[severity]
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'User <username> is authenticated' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'User <first_name> <surname> is authenticated' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        result = client.fetch_current_user()
+        if result.get('_error', None):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            message = f'User {result["user"]["first_name"]} {result["user"]["surname"]} is authenticated'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    demisto.results(message)
+
+    return message
+
+
+def fetch_incidents(client: Client,
+                    last_run: dict,
+                    first_fetch_time: str,
+                    scope: str = None, ) -> tuple[dict[str, str], list[dict]]:
+    """
+    This function will execute each interval (default is 1 minute).
+
+    Args:
+        client: XMCO LePortail client
+        last_run: A dict with a key containing the latest incident created time we got
+        from last fetch
+        first_fetch_time: If last_run is None then fetch all incidents since first_fetch_time
+        scope: filter incident by scope from LePortail
+
+    Returns:
+        next_run: This will be last_run in the next fetch-incidents
+        incidents: Incidents that will be created in Cortex XSOAR
+    """
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get("last_fetch", first_fetch_time)
+    # last_fetch_datetime = datetime.fromisoformat(last_fetch[:-1]).astimezone(timezone.utc)
+    # last_fetch = last_fetch_datetime.strftime(ISO_8601_FORMAT)
+
+    # current_datetime = datetime.utcnow().astimezone(timezone.utc)
+    # current_iso_format_time = current_datetime.strftime("%Y-%m-%dT%H:%M:%S")
+
+    incidents: list[dict[str, Any]] = []
+    items = client.fetch_alert(scope)
+    for item in items:
+        data = item['data']
```

Comment:
Using square brackets can be dangerous. Let's use `.get()` or put each iteration in a `try...catch` block.